### PR TITLE
Android support

### PR DESCRIPTION
**Pull request by CedricGuillemet**
update bgfx and Android support

### CODE REVIEW

**Comment by Hexlord** [comment]
@CedricGuillemet You switched subrepo address in this PR, consider reverting

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "bgfx"]
 	path = bgfx
-	url = https://github.com/bkaradzic/bgfx.git
+	url = https://github.com/syntheticmagus/bgfx.git
```
Comment (widberg) [inline .gitmodules:3]: Please revert this change.

**Comment by widberg** [comment]
I have pulled #58 (Android support and CI). If there are any changes that still need to be made please open a new pull request without changing the bgfx subrepo address.
